.github/workflows/arm-binary.yml

```diff
@@ -0,0 +1,70 @@
+name: Compile Binary for ARM
+
+on:
+  push:
+    branches:
+      - 'staging'
+      - 'trying'
+      - 'master'
+      - 'dev'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        id: toolchain
+        with:
+          profile: minimal
+          override: true
+
+      - name: Cache cargo registry
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/registry
+          # The registry cache is useful as long as we need the same dependencies as another job, regardless of the Rust version and operating system.
+          key: cargo-registry-${{ hashFiles('Cargo.lock') }}-v3
+
+      - name: Cache cargo binaries
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/bin
+          # The cargo binary cache is useful as long as we use the same Rust version and operating system, but regardless of our dependencies.
+          key: ubuntu-latest-cargo-binaries-${{ steps.toolchain.outputs.rustc_hash }}-v2
+
+      - name: Cache target directory
+        uses: actions/cache@v2
+        with:
+          path: target
+          # The target directory is only useful with the same Rust version, dependencies and operating system.
+          key: ubuntu-latest-target-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('Cargo.lock') }}-build-v3
+
+      - name: Setup cross-compilation environment
+        run: |
+          rustup target add armv7-unknown-linux-gnueabihf
```

Comment:
You can install the target also with `actions-rs/toolchain`: https://github.com/actions-rs/toolchain#inputs